adding_species_manuscript.tex

```diff
@@ -613,14 +631,86 @@ \subsection*{Extracting parameters from the literature}
 and to proceed carefully when mixing sources.
 An additional tricky source of inconsistency is coordinate drift between 
 subsequent versions of genome assemblies.
+%REVISION: genetic map --> recombination map (to avoid confusion)
 In \stdpopsim, we follow the UCSC Genome Browser
-and use liftOver to convert the coordinates of genetic maps and genome annotations
+and use liftOver to convert the coordinates of recombination maps and genome annotations
 to the coordinates of the current genome assembly \citep{Hinrichs2006ucsc}.
 
+%REVISION: added section
+\subsection*{Limitations of simulated genomes}
+
+It is important to acknowledge that simulations cannot fully
+capture all aspects of genetic variation as observed in real data,
+with some aspects modeled better than others.
+As mentioned above, this will strongly depend on the demographic model used in simulation.
+Thus, it is important to consider potential limitations of different demographic models
+in reflecting observed genetic variation.
+First, a demographic model inferred from analysis of genomic data will likely depend on
+the samples that contributed the analyzed genomes.
+The inferred demographic model can only reflect the genealogical ancestry
+of these sampled individuals, and this will typically make up a small portion
+of the complete genealogical ancestry of the species.
+% Ilan: I commented out the following statement because I think it's addressed
+% by the final statement in this paragraph. We can uncomment, if needed.
+%(albeit the genealogy of any set of sampled individuals includes many ancestors).
+Thus, demographic models inferred from larger sets of samples from diverse ancestry backgrounds
+may provide a more comprehensive depiction of genetic variation within a species,
+as long as a sufficiently realistic demographic model can be fit.
```

Comment:
Ryan had a good point in the review discussion about substructure and panmixia, and I'm not sure the "sufficiently realistic" caveat quite addresses it, I think out of the context of that discussion it may be too vague. I would be in favor of saying something explicit about substructure and assumed panmixia to anchor the "sufficiently realistic" comment. It complicates things a bit, but now that this is its own section I think that would be worth it for a point that's this important.

Reply:
I think that this can be addressed by adding a comment in parenthesis in the end:
> ... sufficiently realistic demographic model can be fit (models that account for the structure of populations within a species).
It's still a bit vague, but at least we're hinting to the type of realism that is required.
The main point is made in the following sentence - all of this will typically affect only recent events in the inferred demography.